paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/SysRoleControllerTest.java

```diff
@@ -175,4 +179,149 @@ public void testGetRoleList() throws Exception {
                         && ((r.getTotal() > 0 && r.getData().size() > 0)
                                 || (r.getTotal() == 0 && r.getData().size() == 0)));
     }
+
+    @Test
+    @Order(6)
+    public void testChangeRoleStatus() throws Exception {
+        SysRole sysRole = new SysRole();
+        sysRole.setId(2);
+        sysRole.setEnabled(false);
+
+        String responseString =
+                mockMvc.perform(
+                                MockMvcRequestBuilders.put(rolePath + "/changeStatus")
+                                        .cookie(cookie)
+                                        .content(ObjectMapperUtils.toJSON(sysRole))
+                                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                        .accept(MediaType.APPLICATION_JSON_VALUE))
+                        .andExpect(MockMvcResultMatchers.status().isOk())
+                        .andDo(MockMvcResultHandlers.print())
+                        .andReturn()
+                        .getResponse()
+                        .getContentAsString();
+
+        R<Void> r = ObjectMapperUtils.fromJSON(responseString, new TypeReference<R<Void>>() {});
+        assertEquals(200, r.getCode());
```

Comment:
You should assert the data of response to verify whether the response result is expected.